example-doc-generator/python/publish_docs.py

```
@@ -0,0 +1,915 @@
+            "Add it with:\n"
+            f"  git remote add upstream https://github.com/{upstream_slug}.git"
+        )
+
+    if dry_run:
+        dry(f"git fetch upstream  (in {docs_repo})")
+        dry("git checkout dev && git merge upstream/dev --ff-only")
+        dry(f"git checkout -b {branch_name}")
+        return
+
+    info("Fetching upstream/dev...")
+    subprocess.run(["git", "fetch", "upstream"], cwd=str(docs_repo), check=True)
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)